apollo-mpp-utils/src/appleMain/kotlin/com/apollographql/apollo3/mpp/utils.kt

```diff
@@ -33,19 +29,13 @@ actual fun currentThreadName(): String {
 }
 
 actual fun ensureNeverFrozen(obj: Any) {
-  obj.ensureNeverFrozen()
 }
 
-actual fun isFrozen(obj: Any) = obj.isFrozen
-
+actual fun isFrozen(obj: Any) = false
 actual fun freeze(obj: Any) {
-  obj.freeze()
 }
 
 actual fun assertMainThreadOnNative() {
-  check(NSThread.isMainThread()) {
-    "Non-main native call"
-  }
```

Comment:
Something else I forgot to mention: can we have an assert and graceful error message for users that don't use the new MM?